Cargo.toml

```diff
@@ -27,3 +30,4 @@ url = "2.0.0"
 rand = "0.7.0"
 failure = { version = "0.1.5", features = ["derive"] }
 once_cell = "1.0.1"
+http = "0.1.17"
```

Comment:
We would strongly prefer this be an optional dependency, as we avoid shipping the rust HTTP ecosystem (including the `http` crate) on android.

Reply:
Well, that was quick :)

It's mostly involved just to support converting back and forth between HeaderValue and hawk::Header.  Would it make sense to have a different crate to do that?  Or to just make those conversions conditional?

Reply:
I'll do a more thorough review tomorrow, but it came in just as I was checking my email :p.

And I think making the conversions conditional on a `feature` is fine.